src/sections/Projects/GNH/index.js

```diff
@@ -185,6 +186,12 @@ const Projects = () => {
           </Row>
         </div>
       </Container>
+      <div className="callout">
+        <img src={cncf} />
+        <p> Participate in the state of the art. <br/>
+        Join us in the Cloud Native Computing Foundation's Service Mesh Working Group.</p>
+        <Button primary title="Join Us" />
```

Comment:
What url should this button point to?

Reply:
This one will work well - /community/calendar#meetings

Reply:
Done